src/resolvers/Mutation/removeOrganization.ts

```diff
@@ -54,100 +54,47 @@ export const removeOrganization: MutationResolvers["removeOrganization"] =
     superAdminCheck(currentUser!);
 
     // Remove each post and comments associated to it for organization.posts list.
-    organization.posts.forEach(async (postId) => {
-      await Post.deleteOne({
-        _id: postId,
-      });
-
-      await Comment.deleteMany({
-        post: postId,
-      });
-    });
+    await Promise.all(
+      organization.posts.map(async (postId) => {
+        await Post.deleteOne({ _id: postId });
+        await Comment.deleteMany({ post: postId });
+      })
+    );
```

Comment:
Use `deleteMany` function to perform these deletions.

```ts
await Post.deleteMany({
  _id: { $in : { organization.posts }}
})
```

Reply:
then how we will delete the comments per post??

Reply:
await Comment.deleteMany({
 post: { $in: organization.posts }
})